examples/userapi/00_dsl.ipynb

```diff
@@ -0,0 +1,1033 @@
IpfL4c0338TU1JTjzpBZ/E5ylAs0nuel2W4UReHkyZMYHR119Sbod4qj1tqk1y6TyWBychJnz551TLiTm5jXTxzNiiUnhBnBaqpio6g5dmYctFYUaEDrHHeZoxFScbeyZe0qBSPXrNn6zwiMULtJteqkOVHq6KaLJme72A3KRB9aMCCCF6rvZb4SQXekemykzNGOi9YOAo24gE48bPF7+HY7OWhGAm1/fx/9/f0tf/61G+1x9t2h8DwvzTADqhvld7zjHb7+kjUac+/U+ul0GgsLCwiFQrjnnnswPDzsyRN6v+egyQUaKXUlQ5YnJyddcVTJ++q1MDUSaHJh5tRr9zqx0m4Pmh+hLXcCV777x4iGoSrOvGJTxeFRC7c4HrNWjmiFrXK3aiKlFmZLHe2g5qJtmywhBW6XOW4fvq+iSJkSaXLylWoVAhFqdmi1kBA13C7T9HL49p3moHX6z5qPjkBrQTiOQzKZxMLCAmKxGO699150dXXhxRdf9P0JiJ8ChWVZrKysQBRFrK+vOzLbzSp+OmjysI6dnR0kk0kUCgVMTk7WzPJyGoqifEly1BJobggzozXdQs2xMxJsXvfJOUUrlDhGw7VCQL6RV5a/ySnrlPwZIXfe1ISDVvLgpiJe3mnBtluKY0AlLMMt8mykzs0zosLTpsoctw16zghqZY6hAA/28LiyxarIjIZYyy5auzhofogaN4Zvt5uDpnf/z2Qy6Ovra4lr8J1Ee5x9dxilUgm7u7s4e/YsBgcHQVEUKpUKBEHwPXnIjxJHhmGwsLCApaUlKaHy/vvv9yUi1+85aCzL4qWXXkKxWMT09DQmJiY8ucn4MQtNuaabwozQDCWOgL5Ia+WbbDMLS6V7VmRC6Iqou0jy/rNMOYaooi/JqP9su9iNkM3BylpslrohiBSOx2uFmtn+MzXURJpW/9leJV4TyKFGlomiL6wdtGIGK+4ZAOwWq99vJWIfqC1zdIJ2CQlpJtepkeHbLMtic3MTDMOgq6sLkUikZa+tZkscOzQXHYHWgvT19eGBBx6o+Rp5OsKyrK8XSC9LHMvlMtLpNFZWVnDkyBFcvHgRR44cwb/8y7/45mL5ERIiiiK2trbw9ttvg+M4DA8Pe96D6KeD5oUwk6/ppYhQW69QKKBYLKK3t1f1qWirOmjNjtI9k1OohBFX+fuMTk+aGmadnEYgJZJKoaaGVcfKiP1KDEci6mEgWZXwEOW/BYAuldJKvbAQPReNiDMlVssciYsWDArg+arIitAcLn37i3jlf/1vpn5Gx0HzDqPh2/l8Hm+++abkuJVKJSlEQ/lfOBxuauHG8zwEQdDdD+RyuY5Aa0I6Aq1NaIaIe8CbEsdisYh0Oo3V1VXVodvNnqToFKIoYnNzE8lkEgzDYGxsDKlUClNTU57fMPxw0MgT0FdeecV1YUbwcw5aPp9HIpHA1tYWQqGQND+IDIslZTpAcztRWjSzsCTuWZkJIRSt1LhnauLMqjA7qERx4NjRqiMoSvOIUNMSTHpUuNtbB6dLHe26aEUuhP1SHGGbKYpmkZc5arloZTYElgvWJTzq0RFo/kOGb5P7yOzsrHRPLxaLktvWSsO3yX5MT6BlMpnOkOompCPQWhCtzbffCYrkGMrlxkpUtMjn80ilUtjY2MDx48dx9epV9PT01H2f30EdgiC4erMl89ySySQ4jpPGBnAch1Qq5fk8MsBbBy2TySCRSGBvbw+BQAA3btzwbJSEX3PQfvrTn2JjYwNjY2O4du2a9H6TDQOJoc7n8+B5Hj/96U/R29srbRa6u7ul3ooO9gkGBBQZaxsvZXmjkv1S3NS8Myv9Z1YgPW6DsYLtn2FGpB0w1ZJzPRfNDAU2rOqi6aHmoindMzIDjWAnLEQJw9KmXbR2EGjtEHQCVIUmRVHS5xEMBtHT01O332iF4dscxyEQCOieW5lMBoODgx4eVQczdARai6K2UQyFQr4mKALuiKNcLodUKoWtrS2Mjo7i+vXruvM6/BRo5OLrRj+BIAiSMON5HidPnsT4+Li0DjkfOI7zdPYd4I2zRIQZmeE2MTGBN99809PX6qWDViwWJXeUoig89NBDiMfj4HkeDMMgFAqhv7+/pjRFFEW88MILGB8flwTc1taW1Fsh3yyQ/5rlSW+zcuW7fwyg6p4FZUKqUFE/71hB//e+xIYQpVnsl+Ka3+N0/5kZdkpdDYu0WMjc/Ucp0pTljUoXjZQ36v7Mw/eT4WhDF418r91B4HphIaTMMUTzpl00URQhimLLi5t26KMDbic4Gj3QaoXh20b9Z0A1xfGuu+5yZf0O9ukItDbC74h7p48hk8kgmUxid3cX4+PjePjhhxGLGd+o/XbQgOqNyqmNryAIWFtbk9yxmZkZ1XluZG0/ItbdLO1UCjMywy2bzTZNcqSTlEolpFIprK6uYmBgAMFgEHNzc6b+LXnqqxz8Ku+tICU6S0tLNWWScuHm9NBXs8ferCWOZjkoR3R71Qh64sxLeIWYlLtpdvrP9ktxHHGo3FGv1NGOi+YWZsJCjFw0ck1pdXHT6iWOhEYTHJtp+LZRgiPQKXFsVjoCrY1oBget0TJLMrsrlUqpHCunAAAgAElEQVQhk8lgYmICs7OzlhIZ/UxSJJHzTqwvCAJWV1eRSqVAURRmZmYwMjKieRMnm3M/XrsbPWhEoO/t7dUIM/maXgs0N1Mcy+UyUqkUVlZWMDQ0hGvXrgEAXnzxxbpjMDpGpdAhvRWkR43AsqxqmaQgCIjFYjXCzczsoHbj/n/9EkKHd8mghttC+s8OysbXqEwxVueo2HVxrKDsPzNip9SF3oi6OJL3n6lhVqQ1WupoFVLmKBfHnBCoef+dKnOUh4UwLA2e0xdeHYHWXLg1A82P4dssy5py0DoCrfnoCLQWpd1KHJWzuyYmJmyHPvjpoAGNu0mCIGBlZQWpVArBYBB33303RkZGTG2M/UhTdHpdpTCbm5tTPQ/8EGhupDhWKhWk02ksLS3h
```

Comment:
Remove comma and also

Reply:
I'm not sure about this one, there are multiple things impacting the arithmetic complexity of a stencil expression. For example, "also" the derivative order and the finite difference order

Reply:
I think "Which may lead to fairly complex stencils depending on the chosen discretization" or "Which may lead to fairly complex stencils, particularly for high derivative or finite difference orders" scans a little better, but I'm pretty much just nitpicking tbh